spec/deprecation-cop-view-spec.coffee

```diff
@@ -53,13 +53,22 @@ describe "DeprecationCopView", ->
     expect(packageDeprecationItems.eq(2).find("a").attr("href")).toBe(path.join(fakePackageDir, "styles", "old-stylesheet.less"))
 
     jasmine.unspy(atom.packages, 'getLoadedPackages')
+    atom.packages.unloadPackage(pack.name)
 
-  it "updates when themes with deprecated selectors are activated", ->
-    deprecationCopView.find("button.refresh-selectors").click()
-    expect(deprecationCopView.find("button.refresh-selectors")).toBeHidden()
+  it "updates automatically when themes with deprecated selectors are activated", ->
```

Comment:
I updated this test and also use it to verify automatic updates (no calls to `updateSelectors` made and the view refreshes).